src/components/modalPrompt/tests/ModalPromptConnected.spec.tsx

```diff
@@ -0,0 +1,175 @@
+import { mount, ReactWrapper } from 'enzyme';
+import { clearState } from '../../../utils/ReduxUtils';
+import { IReactVaporState } from '../../../ReactVapor';
+import { TestUtils } from '../../../utils/TestUtils';
+import { Store } from 'redux';
+import { Provider } from 'react-redux';
+import { IModalPromptProps, ModalPrompt } from '../ModalPrompt';
+import { ModalPromptConnected } from '../ModalPromptConnected';
+import { openModalPrompt, cancelModalPrompt } from '../ModalPromptActions';
+// tslint:disable-next-line:no-unused-variable
+import * as React from 'react';
+
+describe('ModalPrompt', () => {
+  describe('<ModalPromptConnected />', () => {
+    let modalPrompt: ReactWrapper<IModalPromptProps, any>;
+    let wrapper: ReactWrapper<any, any>;
+    let store: Store<IReactVaporState>;
+    const id: string = 'modalPrompt';
+    const title: string = 'Title';
+    const confirmLabel: string = 'Confirm';
+    const cancelLabel: string = 'Cancel';
+    const message: string = 'message';
+
+    const findInStore = (store: Store<IReactVaporState>, id: string) => {
+      return store.getState().modalPrompts.filter(modalPrompt => modalPrompt.id === id);
+    };
+
+    const findOneInStore = (store: Store<IReactVaporState>, id: string) => {
+      return findInStore(store, id)[0];
+    };
+
+    beforeEach(() => {
+      store = TestUtils.buildStore();
+
+      wrapper = mount(
+        <Provider store={store}>
+          <ModalPromptConnected
+            id={id}
+            title={title}
+            confirmLabel={confirmLabel}
+            cancelLabel={cancelLabel}
+            message={message}
+          />
+        </Provider>,
+        { attachTo: document.getElementById('App') }
+      );
+      modalPrompt = wrapper.find(ModalPrompt).first();
+    });
+
+    afterEach(() => {
+      store.dispatch(clearState());
+      wrapper.unmount();
+      wrapper.detach();
+    });
+
+    it('should get its id as a prop', () => {
+      const idProp = modalPrompt.props().id;
+
+      expect(idProp).toBeDefined();
+      expect(idProp).toBe(id);
+    });
+
+    it('should get its title as a prop', () => {
+      const titleProp = modalPrompt.props().title;
+
+      expect(titleProp).toBeDefined();
+      expect(titleProp).toBe(title);
+    });
+
+    it('should get its message as a prop', () => {
+      const titleProp = modalPrompt.props().message;
```

Comment:
`const messageProp = modalPrompt.props().message`

same thing for other tests under, the const is always `titleProp`